output/schema/schema.json

```diff
@@ -104554,7 +104613,7 @@
         {
           "description": "The analysis function that is used. For example, `count`, `rare`, `mean`, `min`, `max`, or `sum`.",
           "name": "function",
-          "required": true,
+          "required": false,
```

Comment:
`false` is correct here if this interface is only used when creating a job for the first time.  But if this interface is also used to generate code for getting jobs then it needs to be split because the `detectors` in a job retrieved from the get jobs API will always have a `function` set for every detector.